config/routes.rb

```diff
@@ -0,0 +1,9 @@
+Rails.application.routes.draw do
+  
+  resources :messages
+  root 'hello#world'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
```

Comment:
[ OPTIONAL] 
- Kindly remove this commented line of code from your codebase. It is not professional to keep commented codes